index.src.html

```diff
@@ -394,6 +435,233 @@
     </section>
     <section>
       <h2>
+        Synchronizing depth and color video rendering
+      </h2>
+      <p>
+        When rendering, we want to position a color from color video frame to
+        corresponding <a>depth map value</a> or 3D point in space defined by
+        <a>depth map value</a>. We use <a>deprojection distortion
+        coefficients</a> to compensate camera distortion when deprojecting 2D
+        pixel coordinates to 3D space coordinates and <a>projection distortion
+        coefficients</a> in the opposite case, when projecting camera 3D space
+        points to pixels. The algorithm here intents to cover all of the cases.
+        Most commonly, either the depth track has deprojection or the color
+        track has <a>projection distortion coefficients</a>.
+      </p>
+      <p>
+        The <dfn>algorithm to map depth pixels to color pixels</dfn> is as
+        follows:
+      </p>
+      <ol>
+        <li>
+          <a>Deproject</a> <a>depth map value</a> to point in depth 3D space.
+        </li>
+        <li>
+          <a>Transform</a> 3D point from depth camera 3D space to color camera
+          3D space.
+        </li>
+        <li>
+          <a>Project</a> from color camera 3D space to color frame 2D pixels.
+        </li>
+      </ol>
+      <p></p>
+      <section>
+        <h2>
+          <dfn>Deproject</dfn> to depth 3D space
+        </h2>
+        <p>
+          To <a>deproject</a> <a>depth map value</a> to point in depth camera
```

Comment:
Alternative, more concise way to say this (if you won't need to link to the algorithm directly elsewhere in the spec):
`To <dfn>deproject</dfn> a depth map value to point in depth camera 3D space, the user agent must run the following steps:`